src/modules/base/IModule_v1.sol

```diff
@@ -19,9 +19,9 @@ interface IModule_v1 {
 
     /// @notice Module has been initialized.
     /// @param parentOrchestrator The address of the orchestrator the module is linked to.
-    /// @param moduleTitle The title of the module.
+    /// @param metadata The metadata of the module.
     event ModuleInitialized(
-        address indexed parentOrchestrator, string moduleTitle
+        address indexed parentOrchestrator, Metadata metadata
```

Comment:
Structs cant really be represented in the events as far as I know.
If you want to emit the whole metadata struct you have to extend the event to contain every field of it

Reply:
No, they are emitted correctly, I'm using this event in the indexer :P